README.md

```diff
@@ -1,9 +1,9 @@
- ![SwiftCurrent](https://raw.githubusercontent.com/wwt/SwiftCurrent/main/.github/wiki/swiftcurrent-logo.png)
+![SwiftCurrent](https://user-images.githubusercontent.com/79471462/131564417-6f4976f4-270c-41b3-bbe1-428528e2cc2c.png)
 
 <!-- Library Information -->
 [![Supported Platforms](https://img.shields.io/cocoapods/p/SwiftCurrent)](https://github.com/wwt/SwiftCurrent/security/policy)
-[![Swift Package Manager](https://img.shields.io/badge/Swift_Package_Manager-supported-brightgreen)](https://github.com/wwt/SwiftCurrent/wiki/Installation#swift-package-manager)
-[![Pod Version](https://img.shields.io/cocoapods/v/SwiftCurrent.svg?style=popout)](https://github.com/wwt/SwiftCurrent/wiki/Installation#cocoapods)
+[![Swift Package Manager](https://img.shields.io/badge/Swift_Package_Manager-supported-brightgreen)](https://wwt.github.io/SwiftCurrent/installation.html#swift-package-manager)
```

Comment:
Is this actually supposed to be an io link? This looks like it could've been a bad find/replace. Same with the other one. Cause I thought badges needed github.com not the .io links

Reply:
Yes, it is.  This badge is generated with the first link

```markdown
![Swift Package Manager](https://img.shields.io/badge/Swift_Package_Manager-supported-brightgreen)
```

Then we have added a link to the badge so that on clicking it, it will take you to something that makes sense.  We had been redirecting to the installation page which is what this change is continuing.